### PR TITLE
Add wretched pukers

### PR DESCRIPTION
**Pull request by ghost**
### Summary

```SUMMARY: Content "Add `wretched puker` zombies"```

#### Purpose of change

Make an exclusive monster for dumpsites.

#### Describe the solution

The `wretched puker` is an acid-type zombie that is weaker than an `acidic zombie`. It deals very low physical damage and performs `ACID_BARF` special attack. These zombies are very slow, slower than a `decayed zombie` and its eyesight is severely hampered.

Their spawnrate is relatively uncommon, in dumps.

It also upgrades into an `acidic zombie` for maybe more than a month.

#### Describe alternatives you've considered

Wretched pukers are once normal zombies that has developed and produced acidic chemicals from unknown waste products in the dumps. They might have fallen and soaked some of the products into them by accident (zombies, stumbling cuz they dumb).

The strength of that chemical has severely weakened the zombie, as their body wasn't ready to hold the acidic chemical.

#### Additional context

I appreciate the suggestions for this monster :heart:

### CODE REVIEW

**Comment by DaviBones** [comment]
"Retched puker" doesn't make grammatical sense, did you perhaps mean "retching puker" or "wretched puker"?

Retch (verb); past tense: retched, present participle: wretching -- to make the sound and movement of vomiting
Wretched (adjective) -- (of a person) in a very unhappy or unfortunate state

**Comment by Axema** [comment]
I based the name on how it is pronounced. Both `retched` and `wretched` matches this, but to give players a subtle hint, I came up with `retched` :grin:

**Comment by DaviBones** [comment]
```wretched``` and ```retched``` aren't pronounced the same. ```retched``` rhymes with ```fetched``` while ```wretched```... doesn't really have any exact rhymes that I know of, but ```fetid``` is pretty close.

It's your creative license at the end of the day, but it sounds rather strange and jarring to me.

**Comment by Axema** [comment]
Oh... I was reading `retched` as `retch-ed`, and really thought it was rhymed with `wretched`. Well, thanks you for this, now I understand why the name makes no sense :rofl:

**Comment by DaviBones** [comment]
No worries :smile: easy mistake to make. Glad I could help!

**Comment by SirPendrak** [comment]
Zombie looks cool, but arent nowdays only non-mutated zombies allowed to spawn at day one? And then evolve.

**Comment by Axema** [comment]
I don't think it's a good idea... Upgrading a normal zombie into a puker would ruin the spawnrate, as they will appear in the cities.

**Comment by DaviBones** [comment]
You need to [lint your JSONs](http://dev.narc.ro/cataclysm/format.html).

**Comment by Axema** [comment]
It is done :wink:

File: data/json/monsters/monsters.json [hunk 1]
```diff
@@ -3107,6 +3107,54 @@
...
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 2,
+    "volume": "62500 ml",
+    "//": "Higher weight from attached junks.",
+    "weight": 84500,
+    "hp": 55,
+    "speed": 65,
+    "material": [ "flesh" ],
+    "symbol": "Z",
```
Comment (anothersimulacrum) [inline data/json/monsters/monsters.json]:
```suggestion
    "symbol": "Z",
    "looks_like": "mon_zombie_acidic",
```

Comment (Axema) [reply data/json/monsters/monsters.json]: Similar sprite to its upgraded counterpart... But anyways, "someone" might make its own sprite 😄

Comment (anothersimulacrum) [reply data/json/monsters/monsters.json]: Oh, looking at the changes, I didn't realize you already had a looks like for it. There was no need to change it.
Sorry for missing that.

Comment (Axema) [reply data/json/monsters/monsters.json]: It's not a trouble. I've put it in and let it be there now :wink:

**Pull request merged**